### DISCUSSION TITLE
Creating a modal dialog with autoFocus input

### DISCUSSION THREAD

**Discussion by herzaso** [floating-ui/floating-ui#2293] [React DOM] [answered]
I'm trying to implement a modal dialog with an autoFocus input but it doesn't work for me.
I basically used the same dialog example that's in the docs (https://codesandbox.io/s/relaxed-thunder-65bnkm) but added `autoFocus` to the cancel button and it doesn't seem to work.
What am I missing?

Also, in that same example, the dialog is used as a modal dialog (dimming the background) but it doesn't use the `modal` prop in `FloatingFocusManager` - why is that?

**Comment by atomiks** [comment, marked as answer]
`autoFocus` working was only fixed very recently, so you might need to update your package version.

`modal={true}` is the default on `FloatingFocusManager`, so you don't need to specify it.

**Reply by herzaso** [reply]
Thanks. I used 0.22 and just noticed that it was fixed in 0.23
It works now!
